doc/internals/contributing.rst

```diff
@@ -197,6 +197,14 @@ To run JavaScript tests, use ``npm``::
     npm install
     npm run test
 
+.. tip::
+
+  `karma` requires a Firefox binary to use as a test browser.
```

Comment:
Now that I remember, doesn't karma support other browsers?

Reply:
In `karma.conf.js` it is currently set up only as `browsers: ["Firefox"]`

Reply:
I see. Before merging this one, maybe we should have a look at the karma deprecation issue that Jay just opened.

Reply:
I going to merge this for now, because its not clear how long it will take to migrate from karma, and theres no point to leaving this hanging.
Obviously when we migrate, we can/should update this section with any necessary information about how to run/maintain the JS tests

Reply:
I was thinking about this too, but hadn't mentioned it - it would be nice to add Chrome into the mix for JS testing too, and I'll file an issue with a feature request for that soon.